wireguard-install.sh

```diff
@@ -132,6 +132,16 @@ PublicKey = $SERVER_PUB_KEY
 Endpoint = $ENDPOINT
 AllowedIPs = 0.0.0.0/0,::/0" >> "$HOME/$SERVER_WG_NIC-client.conf"
 
+# Ask for pre-shared symmetric key
+read -r -p "Want to use pre-shared symmetric key? [y/N] " response
```

Comment:
It should be at the top of the script with the other `read`.

You can keep a simple if here (so rename your `response` variable.